babe

```diff
@@ -419,6 +492,12 @@ if __name__ == "__main__":
     destroyAppOptionsParser.add_argument('app', help='App name')
     destroyAppOptionsParser.set_defaults(func=destroyCommand)
 
+    #Decommission Cluster
+    decommissionClusterOptionsParser = subparsers.add_parser('decommission', help='Decommission a Mesos slave')
+    decommissionClusterOptionsParser.add_argument('-m', '--mesos-endpoint', help='Mesos master host:port', default='localhost:5050', dest='mesos_endpoint', required=True)
```

Comment:
Why take this? At leat it should not be mandatory. This will in 99% of cases be marathon host:5050. At best this is an override that can be used in the remaining 1% of the cases.